### DISCUSSION TITLE
How do you create the classes for adding the endpoints?

### DISCUSSION THREAD

**Discussion by rawgerjmz** [ardalis/CleanArchitecture#755] [Q&A] [answered]
![image](https://github.com/ardalis/CleanArchitecture/assets/16430530/e33ea1e2-2b9e-47e2-bd7f-56469583d52e)

I know that there are commands by installing the "FastEndpoints.TemplatePack" but I can't find a way to create those as you did or you've created those manually?

**Comment by ardalis** [comment, marked as answer]
I create them manually, typically. Once I have one I just copy it and modify it.
